internal/releasesources/clients.go

```diff
@@ -33,11 +33,11 @@ type openShiftSupportLevelAPIClientInterface interface {
 	getSupportLevels(majorVersion string) (ocpVersionSupportLevels, error)
 }
 
-type releaseGraph struct {
-	Nodes []node `json:"nodes"`
+type ReleaseGraph struct {
```

Comment:
If we are able to remove the wiremock stuff I don't think this should be exported.

Reply:
Looks like it's required for the wiremock stuff in subsystem, this is fine then.